autoload/utils/cmake/fileapi.vim

```diff
@@ -16,11 +16,7 @@ function! s:getReplyFolder(build_dir) abort
 endfunction
 
 function! s:createFile(filename, content) abort
-    new
-    setlocal buftype=nofile bufhidden=hide noswapfile nobuflisted
-    put=a:content
-    execute 'w! ' a:filename
-    q
+    call writefile( [ string( a:content ) ], a:filename )
```

Comment:
This was changed because it created a new buffer and then I had some issues with `wincmd p`.